developer/openshift/dev_setup.sh

```diff
@@ -244,17 +244,14 @@ install_pipeline_service() {
   for app in "pipeline-service" "pipeline-service-storage"; do
     cat << EOF >"$manifest_dir/patch-$app.yaml"
 ---
-apiVersion: argoproj.io/v1alpha1
-kind: Application
-metadata:
-  name: $app
-  namespace: openshift-gitops
-spec:
-  source:
-    repoURL: $GIT_URL
-    targetRevision: $GIT_REF
+- op: replace
+  path: "/spec/sources/0/repoURL"
```

Comment:
take that back, I think the upgrade failure is related to this .... `strict decoding error: unknown field "spec.sources"` shows up in the error message I posted

Reply:
Yes. This is expected. I have changed the gitops operator to `1.8` channel (same as RHTAP) and earlier it was using `stable` channel which deploys 1.5. _Multiple source_ is added to the application CRD in 1.8 only. I am using that feature in this PR to put minio and postgres in one argoCD application `pipeline-service-storage` for dev environments.
I have referred it here last friday.
https://redhat-internal.slack.com/archives/C032EJ007C0/p1686929821478039

Reply:
per that slack thread ^^ @sayan-biswas will confirm the upgrade tests work after we force merge this

Reply:
test now passing ... marking resolved